modules/ROOT/pages/single-sign-on-with-oidc.adoc

```diff
@@ -86,7 +86,7 @@ edit the `authenticationManager-config.properties` as follows:
 Make sure to link:multi-tenancy-and-tenant-configuration#toc2[set the right tenant admin username].
  It is recommended to also replace the value of the passphrase (property `auth.passphrase`) which is used by the engine to verify the authentication request.
  The value must be the same as in the file *bonita-tenant-sp-custom.properties*. +
- If you need users to be able to log in without having an account on the IDP, you can authorize it by setting the property `oidc.auth.standard.allowed` to true. Users will then be able to log in using the portal login page (`/login.jsp`) provided they have a Bonita account and their password is different from their username. +
+ If you need users to be able to log in without having an account on the IDP, you can authorize it by setting the property `oidc.auth.standard.allowed` to true. Users will then be able to log in using the Bonita login page (`/login.jsp`) provided they have a Bonita account and their password is different from their username. +
```

Comment:
provided they have a Bonita account and their password is different from their username. => with the provided Bonita username and a password that is different from it.

Reply:
Ok, this makes more sense